dev/com.ibm.ws.microprofile.openapi/test/com/ibm/ws/microprofile/impl/parser/test/APIResponsesParserTest.java

```diff
@@ -0,0 +1,203 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.impl.parser.test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.ibm.ws.microprofile.openapi.impl.model.PathItemImpl;
+import com.ibm.ws.microprofile.openapi.impl.model.responses.APIResponseImpl;
+import com.ibm.ws.microprofile.openapi.impl.parser.OpenAPIParser;
+import com.ibm.ws.microprofile.openapi.impl.parser.core.models.SwaggerParseResult;
+
+/**
+ *
+ */
+public class APIResponsesParserTest {
+
+    SwaggerParseResult result = new OpenAPIParser().readLocation("airlines_openapi.yaml", null, null);
+
+    @Before
```

Comment:
This method will be run before each test method, which doesn't seem to be needed. Perhaps this should be BeforeClass?